scripts/upload_zoom_recordings.py

```diff
@@ -126,35 +118,21 @@ def get_youtube_client():
             print(f'  Download from {url}...')
             filepath = download_file(url, tmpdirname)
             title = f'{meeting["topic"]} - {pretty_date(meeting["start_time"])}'
+
             # These characters don't work within Python subprocess commands
             chars_to_strip = '<>'
             title = re.sub('['+chars_to_strip+']', '', title)
-            command = [
-                "youtube-upload", filepath,
-                "--title=" + title,
-                "--playlist=" + DEFAULT_YOUTUBE_PLAYLIST,
-                "--category=" + DEFAULT_YOUTUBE_CATEGORY,
-                "--license=" + DEFAULT_VIDEO_LICENSE,
-                "--recording-date=" + fix_date(meeting['start_time']),
-                "--privacy=unlisted",
-                "--client-secrets=client_secret.json",
-                "--credentials-file=.youtube-upload-credentials.json"
-            ]
+
             print('  Adding to main playlist: Uploads from Zoom')
+            video_id = initialize_upload(youtube,
+                              filepath,
+                              title=title,
+                              category=28, # TODO: get category by name: DEFAULT_YOUTUBE_CATEGORY,
```

Comment:
I think we should make this a constant up at the top for now and put the TODO there.